tests/functional/test.lib.imports.js

```diff
@@ -35,19 +35,21 @@ describe('web-ext imported as a library', () => {
   it('can be imported as an ESM module', async () => {
     await withTempDir(async (tmpDir) => {
       execFileSync(npm, ['link', 'web-ext'], {cwd: tmpDir.path()});
-      shell.cp('-rf', `${fixtureEsmImport}/*`, tmpDir.path());
+      shell.cp('-rf', `${fixturesUseAsLibrary}/*`, tmpDir.path());
       execFileSync(node, ['--experimental-modules', 'test-import.mjs'], {
         cwd: tmpDir.path(),
+        stdio: 'inherit',
```

Comment:
Nit, do we need to keep `stdio: 'inherit'` here (and at line 52)? if I recall correctly the child process stderr would still be echoed by the parent process and so the assertion errors would be visible in case of failure (and we don't log anything to stdout that we really need to see, besides while we were investigating the windows issue and we wanted to see a console.log from the child process as part of the tests output).

If that is the case let's remove `stdio: 'inherit'`.

Reply:
Yep, you are right, removing it.